internal/adapter/fzf/fzf.go

```diff
@@ -72,6 +74,8 @@ func New(opts Opts) (*Fzf, error) {
 		opts.Delimiter = "\x01"
 	}
 
+    defaultFzfArgs := strings.Split(os.Getenv("FZF_DEFAULT_OPTS"), " ")
```

Comment:
Here and also line 96, you can use `shellquote.Split()` instead, this will make sure that if one term is between quotes it won't be split at the inner space characters.

It returns an error, in Go you need to check manually if the error is null, and return if needed:

```suggestion
    rawDefaultOpts := os.Getenv("FZF_DEFAULT_OPTS")
    defaultFzfArgs, err := shellquote.Split(rawDefaultOpts)
	if err != nil {
		return nil, errors.Wrapf(err, "can't split the fzf options: %s", rawDefaultOpts)
	}
```